src/algebra/module/localized_module.lean

```diff
@@ -33,7 +33,6 @@ Given a commutative ring `R`, a multiplicative subset `S ⊆ R` and an `R`-modul
 ## Future work
 
  * Redefine `localization` for monoids and rings to coincide with `localized_module`.
```

Comment:
Something that is has to be done is to generalize the API for `localized_module` to anything that satisfies the characteristic predicate.

Reply:
ibid

Reply:
In #15584, Eric is trying to make `localized_module` and `localization` definitionally equal